rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
 channel = "nightly-2024-04-22"
 components = ["rust-src"]
-targets = ["wasm32-unknown-unknown"]
+targets = ["x86_64-apple-darwin"]
```

Comment:
Why did you modify this completely unrelated properly?

Reply:
my mistake, sorry 🙏🏽. I will remove it in the squashed commit. Also, I will check that I do not have any irrelevant changes in the commit.